README.md

```diff
@@ -1,5 +1,9 @@
 # Percona Query Analytics Agent
 
+[![Travis CI Build Status](https://travis-ci.org/percona/qan-agent.svg?branch=master)](https://travis-ci.org/percona/qan-agent)
+[![GoDoc](https://godoc.org/github.com/percona/qan-agent?status.svg)](https://godoc.org/github.com/percona/qan-agent)
+[![Report Card](http://goreportcard.com/badge/percona/qan-agent)](http://goreportcard.com/report/percona/qan-agent)
+
```

Comment:
I've added some badges for showing of quality of the package and to link to some tools